docs/overview/index.mdx

```diff
@@ -103,158 +115,290 @@ and offerings.
-				height="24"
-				viewBox="0 96 960 960"
-				width="24"
-			>
-				<path d="M766 936q-41 0-71.5-24.5T656 852H443q-66 0-109.5-43.5T290 699q0-66 43.5-109.5T443 546h77q41 0 67-26t26-67q0-41-26-67t-67-26H304q-9 35-39 59.5T194 444q-48 0-81-33t-33-81q0-48 33-81t81-33q41 0 71 24.5t39 59.5h216q66 0 109.5 43.5T673 453q0 66-43.5 109.5T520 606h-77q-41 0-67 26t-26 67q0 41 26 67t67 26h213q9-35 39-59.5t71-24.5q48 0 81 33t33 81q0 48-33 81t-81 33ZM194 384q23 0 38.5-15.5T248 330q0-23-15.5-38.5T194 276q-23 0-38.5 15.5T140 330q0 23 15.5 38.5T194 384Z" />
-			</svg>
-			<p>
-				Coming from a previous version? Check out our <a>upgrade guide</a>.
-			</p>
-		</div>
-	</Link>
-</div>
 
-## Secure
+## Modules
+
+Once ngrok is in front of your app, you can add authentication, acceleration, transformation, and other behaviors.
 
 <div className="ngrok--cards ngrok--cards-row">
-	<Link to={`/docs/guides/securing-your-tunnels`}>
+	<Link to={`/http/basic-auth/`}>
 		<div className="ngrok--card">
 			<div className="ngrok--card-heading">
-				<h3>Securing your tunnels</h3>
+				<h3>Basic Auth</h3>
 			</div>
-			<p>This guide will walk you through running and using ngrok securely →</p>
+			<p>Require a username and password with HTTP Basic Auth.</p>
+			<a>Read more →</a>
 		</div>
 	</Link>
-	<Link
-		to={`https://blog.ngrok.com/posts/how-to-secure-your-network-tunnels-with-oauth-fast`}
-	>
+	<Link to={`/http/ip-restrictions/`}>
 		<div className="ngrok--card">
 			<div className="ngrok--card-heading">
-				<h3>OAuth</h3>
+				<h3>IP Restrictions</h3>
 			</div>
-			<p>
-				Secure your application using OAuth. Up to 50 MAUs on the free tier →
-			</p>
+			<p>Allow or deny traffic based on the source IP of connections.</p>
+			<a>Read more →</a>
 		</div>
 	</Link>
 </div>
+
 <div className="ngrok--cards ngrok--cards-row">
-	<Link to={`/docs/cloud-edge/modules/webhook-verification/`}>
+	<Link to={`/http/oauth/`}>
+		<div className="ngrok--card">
+			<div className="ngrok--card-heading">
+				<h3>OAuth</h3>
+			</div>
+			<p>Enforce an OAuth flow to well-known IdPs like Google.</p>
+			<a>Read more →</a>
+		</div>
+	</Link>
+	<Link to={`/http/webhook-verification/`}>
 		<div className="ngrok--card">
 			<div className="ngrok--card-heading">
 				<h3>Webhook Verification</h3>
 			</div>
-			<p>Secure your webhooks. Up to 500 verifications on the free tier →</p>
+			<p>Verify HTTP requests are signed by a webhook provider like Slack or GitHub.</p>
+			<a>Read more →</a>
 		</div>
 	</Link>
 </div>
 
-#### Enterprise Only
-
 <div className="ngrok--cards ngrok--cards-row">
-	<Link to={`/docs/cloud-edge/modules/mutual-tls/`}>
-		<div className="ngrok--card ngrok--card-sm">
+	<Link to={`/http/request-headers/`}>
+		<div className="ngrok--card">
 			<div className="ngrok--card-heading">
-				<h4>mTLS</h4>
+				<h3>Request Headers</h3>
 			</div>
+			<p>Add or remove headers from HTTP requests before they are sent to your upstream service.</p>
```

Comment:
<p>Add or remove headers from HTTP requests before they are sent to your upstream service</p>